peps/pep-0729.rst

```diff
@@ -0,0 +1,414 @@
+  resources to fully implement it or if they believe a different behavior better
+  serves their users. However, they should document such deviations from the
+  spec.
+* For projects such as typeshed, or libraries that want to be compatible with
+  multiple type checkers, it provides a set of rules that they can follow to
+  make their code understood by type checkers.
+* For people who want to propose changes to the type system, it provides a
+  foundation for any new proposals.
+
+Notably, the specification is not aimed at end users of typing. End users
+are better served by a more informal user-facing reference, which is discussed
+in the next section.
+
+There are different opinions within the community about how formal such a
+specification should be. While this document recommends an incremental
+approach that builds off existing specification, it does not aim to
+prescribe a final state. The Typing Council would provide a mechanism
+to allow the specification to evolve to meet the level of formality that
+the community desires, for instance, by incorporating parts of
+Kevin Millikin's `document on "Python Static Types" <https://docs.google.com/document/d/1mVCU-nVPT_zVfqivVdMY1aOOZqJ9lsgOLweO1U3uwUM/edit>`_
+as a means to achieve a better formalisation of the spec.
+
+Proposed changes to the specification, including PEPs, should
+generally be accompanied by the following:
+
+* Changes to the conformance test suite that demonstrate the
+  specified behavior.
+* Buy-in from type checker maintainers to confirm that the
+  change can be implemented and maintained within their type
+  checkers.
+* For changes to existing features, a survey of the behavior
+  of existing type checkers. If existing type checkers
+  behave roughly similarly, that is evidence that their shared
+  behavior should be made part of the specification.
+
+User-facing reference for the type system
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Documentation is important for the success of the Python type system, so
+the Typing Council should ensure that there is good documentation for the
+type system.
+
+As mentioned previously, PEPs are point in time change proposals aimed at
+multiple audiences that are hard to clarify. This makes them ill-suited as user
+documentation. The specification discussed in the previous section would
+be a living document, but it would likely be too technical to serve as
+documentation for normal usage.
+
+Therefore, a separate user-facing reference for the type system would be
+useful. Such an effort could expand the documentation on
+`typing.readthedocs.io <https://typing.readthedocs.io/en/latest/>`_ and
+reuse material from the documentation sections of individual type checkers
+and the CPython documentation.
+
+Amendments
+----------
+
+This PEP serves as a charter for the Typing Council. Changes to its operation
+can be made either through a new PEP or through a change to this PEP. In either
```

Comment:
I would prefer updating this PEP. When would we need a new PEP?

Reply:
I suppose if someone in the future decides that this whole Typing Council is a terrible idea and we should throw it away and do something totally different. I hope that's unlikely, but it's worth acknowledging as a possibility.

Reply:
Sure, but that applies to *anything* codified in a PEP. I'm fine with keeping the wording you have now though.